### PR TITLE
add user agent in getusermembership

### PR DESCRIPTION
**Pull request by yungez**
add user agent in getusermembership

@ZhijunZhao @yaweiw @aovechki

### CODE REVIEW

File: activedirectory/azure-ad-integration-spring-boot-autoconfigure/src/main/java/com/microsoft/azure/autoconfigure/aad/AzureADGraphClient.java [hunk 1]
```diff
@@ -14,17 +14,24 @@
...
 
+import com.microsoft.azure.spring.common.GetHashMac;
+
 public class AzureADGraphClient {
 
+    private static String userMembershipRestAPIv1 = "https://graph.windows.net/me/memberOf?api-version=1.6";
+    private static String DEFAULT_USER_AGENT = "spring-autoconfigure/0.1.6-beta";
+
     public static String getUserMembershipsV1(String accessToken) throws Exception {
         final String userMembershipRestAPIv1 = "https://graph.windows.net/me/memberOf?api-version=1.6";
```
Comment (ZhijunZhao) [inline activedirectory/azure-ad-integration-spring-boot-autoconfigure/src/main/java/com/microsoft/azure/autoconfigure/aad/AzureADGraphClient.java]: duplicated?

File: activedirectory/azure-ad-integration-spring-boot-autoconfigure/src/main/java/com/microsoft/azure/autoconfigure/aad/AzureADGraphClient.java [hunk 2]
```diff
@@ -14,17 +14,24 @@
...
         final URL url = new URL(userMembershipRestAPIv1);
+        final String hashMac = GetHashMac.getHashMac();
 
         final HttpURLConnection conn = (HttpURLConnection) url.openConnection();
         // Set the appropriate header fields in the request header.
-        conn.setRequestProperty("api-version", "1.6");
+        conn.setRequestProperty("api-version", "0.1.6");
```
Comment (ZhijunZhao) [inline activedirectory/azure-ad-integration-spring-boot-autoconfigure/src/main/java/com/microsoft/azure/autoconfigure/aad/AzureADGraphClient.java]: what's the difference between 

api-version=1.6 in the userMembershipRestAPIv1 url

and 

conn.setRequestProperty("api-version", "0.1.6");

?

File: activedirectory/azure-ad-integration-spring-boot-autoconfigure/src/main/java/com/microsoft/azure/autoconfigure/aad/AzureADGraphClient.java (hunk 1 above)
Comment (yungez) [inline activedirectory/azure-ad-integration-spring-boot-autoconfigure/src/main/java/com/microsoft/azure/autoconfigure/aad/AzureADGraphClient.java]: merge issue, will fix

File: activedirectory/azure-ad-integration-spring-boot-autoconfigure/src/main/java/com/microsoft/azure/autoconfigure/aad/AzureADGraphClient.java (hunk 2 above)
Comment (yungez) [inline activedirectory/azure-ad-integration-spring-boot-autoconfigure/src/main/java/com/microsoft/azure/autoconfigure/aad/AzureADGraphClient.java]: try to set version to align with package version. revert back

**Comment by yungez** [comment]
still working with bi team to check weather this is right way to set user agent, close this pr for now.

**Pull request closed without merging**